autoload/go/fmt.vim

```diff
@@ -69,6 +69,7 @@ function! go#fmt#Format(withGoimport) abort
     let bin_name = "goimports"
   endif
 
+  let cursor_pos = getpos(".")
```

Comment:
You can use `col(".")` which returns the column directly. Also I think it's better to change the variable to `let current_col`